doc/nrf/known_issues.rst

```diff
@@ -155,6 +155,30 @@ Thread
 
 .. rst-class:: v1-4-0
 
+KRKNWK-6848: Reduced throughput observed.
+  Performance testing for NCP sample shows 20 % decreased throughput with regards with vanilla OpenThread, passing from about 110 kbps to about 89 kbps.
```

Comment:
86 kbps not 89

Reply:
KRKNWK-6848 says 89. However, I'll remove the specific data.